website/docs/tutorial/chat-termination.ipynb

```diff
@@ -35,7 +35,7 @@
    "source": [
     "import os\n",
     "\n",
-    "from autogen import ConversableAgent"
+    "from autogen.agentchat import ConversableAgent"
```

Comment:
This change is unnecessary and makes the line longer. I suggest not making this change.

Reply:
sure, we can close this PR.